src/contributors.njk

```diff
@@ -0,0 +1,15 @@
+---json
+{
+  "layout": "layout"
+}
+---
+
+<h1>Contributors</h1>
+
+<div class="contributors">
+{% for contributor in github.contributors %}
+  <a href={{ contributor.url }}>
+    <img alt="Profile picture for {{ contributor.login }}" class="thumbnail" src="{{ contributor.avatar_url }}" />
```

Comment:
Add the `contributor.login` underneath the thumb nail please

Reply:
There are some quite long usernames so this doesn't look very nice :(

Reply:
Hmm fair enough, I guess they could be truncated over 20 chars?